components/glossary.js

```diff
@@ -136,6 +136,21 @@ export default function Glossary() {
                 </>
               }
             />
+            <GlossaryItem
+              question="I have more questions."
+              answer={
+                <>
+                  Contact us! Reach out on #epoch on Slack or email us at {" "}
```

Comment:
```suggestion
                  Contact us! Reach out on #epoch on the Hack Club Slack or email us at {" "}
```
Could we also link https://hackclub.com/slack